g1: starting
Yielded: y1
g1: about to yield from g2
g2: starting
Yielded: y2
g2: about to yield from g1
Traceback (most recent call last):
  File "test19.py", line 20, in <module>
    for y in gi:
  File "test19.py", line 16, in g2
    yield from gi
  File "test19.py", line 9, in g1
    yield from g2()
ValueError: generator already executing
